woocommerce-services.php

```diff
@@ -1614,9 +1614,9 @@ public function enqueue_wc_connect_script( $root_view, $extra_args = array() ) {
 			$escaped_arguments = function_exists( 'wc_esc_json' ) ? wc_esc_json( $encoded_arguments ) : esc_attr( $encoded_arguments );
 
 			?>
-				<div class="wcc-root woocommerce <?php echo esc_attr( $root_view ); ?>" data-args="<?php echo $escaped_arguments; ?>">
+				<div class="wcc-root woocommerce <?php echo esc_attr( $root_view ); ?>" data-args="<?php echo $escaped_arguments; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>">
```

Comment:
`wc_esc_json` is present since `3.5.5` and WCS&T currently requires `3.6` so we can just use it.

```suggestion
				<div class="wcc-root woocommerce <?php echo esc_attr( $root_view ); ?>" data-args="<?php echo wc_esc_json( $encoded_arguments ); ?>">
```

And then the `$escaped_arguments` variable can be removed.